jupyter_foundation.md

```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.
+
+
+## Jupyter Foundation Governing Board
+
+The Jupyter Foundation Governing Board controls the resources of the Jupyter Foundation and works to build a healthy collaboration between the Jupyter community, Jupyter leadership, and members of the Jupyter Foundation, including for-profit organizations. The current members of the Jupyter Foundation Governing Board are listed in the [Leadership Directory](people.md#jupyter-foundation-governing-board).
+
+## Responsibilities of the Jupyter Foundation Governing Board
+
+- Serve Project Jupyter and its mission.
+- Meet regularly to review strategic objectives of Jupyter.
+- Develop and approve an annual strategic plan and budget for funding from the Jupyter Foundation to benefit Project Jupyter.
+- Recruit and retain Jupyter Foundation members.
```

Comment:
Not sure about the ideal wording, but I think we should explicitly list budgetary decisions/allocations on other potential timescales. I don't imagine that a single annual plan/budget would be the only mechanism for allocating funds we should have.

Perhaps something like 

> "Meet at least quarterly and allocate budgetary resources from the fund as appropriate, based on both the annual strategic plan and potential needs and opportunities that may arise and be brought up to the Board for consideration". 

Probably too wordy, but it's a start...